github_utils.py

```diff
@@ -81,7 +82,7 @@ def new_crash_with_backoff(cls, crash_report):
         """
         there is a chance that we get a new crash before an issue was submitted before.
         """
-        backoff_cache_key = cls.backoff_cache_key(crash_report)
+        backoff_cache_key = cls.backoff_crash_key_new_crash(crash_report)
```

Comment:
Yes, i refactored the name so we use different backoff keys for comments and new issues.